src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -85,6 +86,11 @@ public function getConfigTreeBuilder()
                 ->scalarNode('ide')->defaultNull()->end()
                 ->booleanNode('test')->end()
                 ->scalarNode('default_locale')->defaultValue('en')->end()
+                ->arrayNode('enabled_locales')
+                    ->info('A list of enabled locales. Allows to determine the user\'s locale according to the Accept-Language header.')
+                    ->prototype('scalar')->end()
+                    ->defaultValue([])
```

Comment:
this is useless. A prototyped array node already defaults to the empty array without configuring it explicitly.